src/_gettsim_tests/test_functions_loader.py

```diff
@@ -7,7 +7,7 @@
 import pytest
 
 from _gettsim.config import RESOURCE_DIR
-from _gettsim.functions_loader import (
```

Comment:
This will need updating for the new structure.

E.g., `_load_functions` is left in `functions.preprocessor` but should actually be deleted because it has moved to `functions.loader`. 

In the course of that, we should talk about the structure of those two modules. E.g., I would not expect a function `load_and_check_functions` in `functions.preprocessor`, I would have expected that as part of the setup of the `PolicyEnvironment` already? But I guess that one does not know about the targets yet?

We may also leave as is for the moment and do a quick cleanup PR ex post. But I think clearing up a little more would help for the changes ahead. Let's discuss.